src/EnergyPlus/RefrigeratedCase.cc

```diff
@@ -6226,7 +6231,8 @@ void GetRefrigerationInput(EnergyPlusData &state)
                 TransSystem(TransRefrigSysNum).SumUASuctionPipingLT = Numbers(4);
                 TransSystem(TransRefrigSysNum).SuctionPipeActualZoneNumLT =
                     UtilityRoutines::FindItemInList(Alphas(AlphaNum), state.dataHeatBal->Zone);
-                TransSystem(TransRefrigSysNum).SuctionPipeZoneNodeNumLT = DataZoneEquipment::GetSystemNodeNumberForZone(state, Alphas(AlphaNum));
+                TransSystem(TransRefrigSysNum).SuctionPipeZoneNodeNumLT =
+                    DataZoneEquipment::GetSystemNodeNumberForZone(state, TransSystem(TransRefrigSysNum).SuctionPipeActualZoneNumLT);
```

Comment:
Now I see why you went to the trouble of changing the string arg to an int. This function is used quite a lot.